cfme/tests/test_rest.py

```diff
@@ -374,13 +374,446 @@ def test_vm_add_lifecycle_event(request, setup_a_provider, rest_api, vm, from_de
 }
 
 
+@pytest.fixture(scope="module")
+def users_data():
+    name = fauxfactory.gen_alphanumeric()
+    result = [{
+        "name": "name_{}".format(name),
+        "userid": "userid_{}".format(name),
+        "email": "{}@local.com".format(name),
+    } for _ in range(1, 5)]
+
+    return result
+
+
+@pytest.fixture(scope="module")
+def groups_data():
+    name = fauxfactory.gen_alphanumeric()
+    result = [{
+        "description": "{}".format(name),
+        "miq_user_role_id": _,
+    } for _ in range(1, 5)]
+
+    return result
+
+
+@pytest.fixture(scope="module")
+def roles_data():
+    name = fauxfactory.gen_alphanumeric()
+    result = [{
+        "name": "{}".format(name),
+        "settings": {
+            "restrictions": {
+                "vms": "user"
+            },
+        },
+    } for _ in range(1, 5)]
```

Comment:
Then please name the `_` as something, since `_` looks like you don't need that value any more.